go.mod

```diff
@@ -1,18 +1,18 @@
 module github.com/openshift/pagerduty-operator
 
-go 1.21
+go 1.24.6
```

Comment:
Technically this should be the minimum version this project supports, so setting this so explicitly means you need 1.24.6 or newer to compile. I would leave this as say 1.23 or something.

By nature of how boilerplate and operator builds work, you will be built with Go 1.24.z, whichever is in ubi9 when we built the boilerplate image.